src/components/v1/shared/BudgetForm.tsx

```diff
@@ -271,7 +272,9 @@ export default function BudgetForm({
             style={{ marginTop: 20 }}
             htmlType="submit"
             type="primary"
-            onClick={() => onSave(currency, target, duration)}
+            onClick={() =>
+              onSave(getV1CurrencyOption(currency), target, duration)
```

Comment:
I'm wondering if we need this function 🤔 

Could we instead use currency context here?

Reply:
It's a version-specific component so it's not really necessary I don't think